packages/api/src/contracts/user.repository.service.ts

```diff
@@ -0,0 +1,15 @@
+import { RegisteredUser, Survey, User, UserInformation } from '@coderscamp/shared/models/user';
+
+export abstract class UserRepositoryService {
```

Comment:
Nice abstraction over Prisma - good :) 
But, This name... "UserRepository" is enough. Service / Repository. 

More: Repository should be per entity. For user or for survey - no for both. 
You don't need relation. If everything is related to user and doing by mean of user, then you will end up with methods only about user. 

I necourage you to read first email from: 
https://zycienakodach.pl/lista-mailingowa